Update Defender for Cloud Privacy Section to include GDPR DSR Export Instructions for Defender for APIs (#109811)
Azure/azure-content: articles/defender-for-cloud/privacy.md

> ## Next steps

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

> [What is Microsoft Defender for Cloud?](defender-for-cloud-introduction.md)

Reviewer: ```suggestion
```

> 

Reviewer: ```suggestion
```

## Respond to data subject export requests for Defender for APIs
The right of data portability allows data subjects to request a copy of their personal data in a structured, common, electronic format that can be transmitted to another data controller.

### Manage export and view requests

> Manage requests to export customer or user data.

Reviewer: ```suggestion
You can manage requests to export customer or user data.
```

How do you do this though?

> #### Export customer data (tenant admin)

Reviewer: ```suggestion
#### Export customer data (Tenant administrator only)
```

> As a tenant administrator, export customer data.

Reviewer: ```suggestion
As a tenant administrator, you have the ability to export customer data.
```

> 

Reviewer: ```suggestion

**To export customer data**:
```

> 1. Send an email to D4APIS_DSRRequests@microsoft.com specifying the customer’s email address in the request. The Defender for APIs team will send an email to the registered tenant admin email address, asking for confirmation to export data.

Reviewer: ```suggestion
1. Send an email to `D4APIS_DSRRequests@microsoft.com` that specifies the customer’s email address in the request. 

    The Defender for APIs team respond with an email to the registered tenant's administrator email address that will ask for confirmation to export the data.
    
## Next steps

[What is Microsoft Defender for Cloud?](defender-for-cloud-introduction.md)

```

> 2. Acknowledge the confirmation to export the data for the requested customer.

Reviewer: ```suggestion
1. Acknowledge the confirmation to export the data for the requested customer.
```

> 3. Receive the exported data through the tenant admin email address.

Reviewer: ```suggestion
The exported data will be sent to the tenant administrator's email address.
```
